cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-media.php

```diff
@@ -1812,7 +1812,7 @@ public function setup() {
 	 * @return array
 	 */
 	public function match_file_name_with_cloudinary_source( $image_meta, $attachment_id ) {
-		if ( $this->has_public_id( $attachment_id ) ) {
+		if ( is_array( $image_meta ) && isset( $image_meta['file'] ) && $this->has_public_id( $attachment_id ) ) {
```

Comment:
@DavidCramer 

This is ok, but you could have used instead `! empty( $image_meta['file'] ) && `